### PR TITLE
UI specific changes on auth page

### PR DESCRIPTION
**Pull request by hrachocode**
Make UI-specific changes on Auth sign-up/sign-in pages on different components compared with Figma.  changes are pixel-based so I cant provide screenshots this time.

 1. Font sizes
 2. paddings
 3. heights/widths
 4. colors
 
 followed the material ui design system.

### CODE REVIEW

File: libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx [hunk 1]
```diff
@@ -19,10 +18,16 @@ export const GenericFormCancelButton = ({
   return (
     <StyledGenericFormCancelButton>
       <Button
+        sx={{
+          background: '#ffffff',
```
Comment (alexworker23) [inline libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx]: Please, don't use hard-coded colors, where possible. 
This color is existing in `variables` so you can use it from there

File: libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx [hunk 2]
```diff
@@ -3,12 +3,11 @@ import { Button } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
 import { Add } from '@material-ui/icons';
 import { useStyles } from './generic-form-cancel-button.styles';
-
+import { variables } from '@energyweb/zero-theme';
 /* eslint-disable-next-line */
```
Comment (alexworker23) [inline libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx]: I think we can remove this disabling comment

File: libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx [hunk 3]
```diff
@@ -7,17 +7,20 @@ import { GenericFormContext } from '../../providers';
 import { useContext } from 'react';
 
 /* eslint-disable-next-line */
-export interface GenericFormSubmitButtonProps {}
+export interface GenericFormSubmitButtonProps {
+  name: string;
+}
 
 const StyledGenericFormSubmitButton = styled.div``;
```
Comment (alexworker23) [inline libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:13]: What's the use of it if there is no styling ?

File: libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx [hunk 4]
```diff
@@ -26,7 +29,9 @@ export const GenericFormSubmitButton = () => {
         color={'primary'}
         endIcon={<ChevronRight className={styles.icon} />}
       >
-        {t('forms.submitBtnText')}
+        {prop.name === 'sign-in'
+          ? t('forms.signInBtnText')
+          : t('forms.signUpBtnText')}
```
Comment (alexworker23) [inline libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:33]: So now generic submit form button will have either "Sign In" or "Sign Up" text, and won't be used a regular form submit button?

File: libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx [hunk 5]
```diff
@@ -7,17 +7,20 @@ import { GenericFormContext } from '../../providers';
 import { useContext } from 'react';
 
 /* eslint-disable-next-line */
```
Comment (alexworker23) [inline libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx]: can be removed as well

Comment (hrachocode) [reply libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx]: sure, will remove, just decided not to touch something that is not my change.

File: libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx (hunk 3 above)
Comment (hrachocode) [reply libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:13]: sorry you mean my change or empty styled.div?

File: libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx (hunk 2 above)
Comment (hrachocode) [reply libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx]: sure, will remove, just decided not to touch something that is not my change

File: libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx (hunk 1 above)
Comment (hrachocode) [reply libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx]: sorry, will replace this with variable as well.

File: libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx (hunk 4 above)
Comment (hrachocode) [reply libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:33]: I think its not. but before it was using only one text, I followed figma and make it two, sign in and sign up. but ideally it should be generic and should use that text as an argument

File: libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx (hunk 3 above)
Comment (alexworker23) [reply libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:13]: empty styled.div

Comment (hrachocode) [reply libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:13]: this empty-styled. div is used to get wrapped with another content. and this structure is followed on other files/folders too. not sure if this is a good idea to remove it.

File: libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx (hunk 4 above)
Comment (alexworker23) [reply libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:33]: @hrachocode ok, so what is the solution for this button?

Comment (hrachocode) [reply libs/ui/core/src/lib/components/generic-form-submit-button/generic-form-submit-button.tsx:33]: I think we should understand what's the purpose and perspective usage of this button and make it more reusable with another task/pr, current PR is only related to making the login/sign-up page similar to Figma.

**Review by alexworker23 — APPROVED** [review]

**Pull request merged**